src/main/java/com/artipie/http/ArtipieRepositories.java

```diff
@@ -67,7 +68,7 @@ public ArtipieRepositories(final Settings settings) {
     public Slice slice(final Key name, final boolean standalone) {
         final Storage storage = this.settings.storage();
         return new AsyncSlice(
-            storage.exists(new Key.From(String.format("%s.yaml", name.string()))).thenCompose(
+            new ConfigFilesExtensions(name.toString()).exists(storage).thenCompose(
```

Comment:
@genryxy again here "ConfigFilesExtensions" checking that "exists" does not make sense to me, I'd suggest to make class redesign